meta-facebook/at-cb/src/ipmi/plat_ipmi.c

```diff
@@ -145,16 +145,18 @@ void OEM_1S_GET_FW_VERSION(ipmi_msg *msg)
 
 		if (pex_access_engine(cfg->port, cfg->target_addr, p->idx, pex_access_sbr_ver,
 				      &reading)) {
-			if (cfg->post_sensor_read_hook(cfg, cfg->post_sensor_read_args, NULL) ==
-			    false) {
-				LOG_ERR("PEX%d post-read failed!",
-					component - CB_COMPNT_PCIE_SWITCH0);
+			if (cfg->post_sensor_read_hook != NULL) {
+				if (cfg->post_sensor_read_hook(cfg, cfg->post_sensor_read_args,
+							       NULL) == false) {
+					LOG_ERR("PEX%d post-read failed!",
+						component - CB_COMPNT_PCIE_SWITCH0);
+				}
 			}
 			msg->completion_code = CC_PEX_ACCESS_FAIL;
 			return;
 		}
 
-		if (cfg->post_sensor_read_hook) {
+		if (cfg->post_sensor_read_hook != NULL) {
```

Comment:
I don't think u need this != NULL  check, if(x) should already check for it.

Reply:
Hi @jagpalgill ,
Modify with your suggestion, please help to review again, thank you.

Regards,
Lora.